<ui version="4.0" >
 <class>configElement</class>
 <widget class="QWidget" name="configElement" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>143</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="sizeIncrement" >
   <size>
    <width>234</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QLabel" name="variableName" >
       <property name="text" >
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="iconLabel" >
         <property name="text" >
          <string/>
         </property>
         <property name="pixmap" >
          <pixmap resource="../resources.qrc" >:/icons/start_ally.xpm</pixmap>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QLineEdit" name="lineEdit" />
       </item>
       <item row="0" column="2" >
        <widget class="QToolButton" name="fileBrowserToolButton" >
         <property name="text" >
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="descriptionLabel" >
     <property name="text" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:10pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:9pt;">&lt;span style=" font-size:10pt;">this text is &lt;/span>&lt;span style=" font-size:10pt; font-weight:600;">rich ;-)&lt;/span>&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-weight:600;">and we love qt!&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-weight:600;">Third row.&lt;/p>&lt;/body>&lt;/html></string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc" />
 </resources>
 <connections/>
</ui>
